analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -522,14 +522,43 @@ def process_config_file(args):
 
     cfg = load_json_or_empty(args.config_file, default={})
     if cfg.get("enabled"):
-        print("Using config file: '{0}'.".format(args.config_file))
+        LOG.debug("Using config file: '{0}'.".format(args.config_file))
```

Comment:
Logger is not initialized here (it will be initialized in the main), so do not use it.